.github/workflows/ci-testing.yml

```diff
@@ -93,5 +93,5 @@ jobs:
         uses: actions/upload-artifact@v4
         if: failure()
         with:
-          name: image-failures-${{ matrix.os }}-${{ matrix.python-version }}
-          path: cartopy_test_output-${{ matrix.os }}-${{ matrix.python-version }}
+          name: image-failures-${{ matrix.os }}-${{ matrix.python-version }}-${{ matrix.shapely-dev }}
+          path: cartopy_test_output-${{ matrix.os }}-${{ matrix.python-version }}-${{ matrix.shapely-dev }}
```

Comment:
This needs to match line 82.

Reply:
Many thanks! I had not thought to look anywhere else.

7a58dc72b94a7d21d3e9467536882783a8ee889f